### ISSUE TITLE
ir receiver / passtrough

### ISSUE THREAD

**Issue by FDelporte** [wouterverweirder/home-remote#5]
add infrared receiver + code to simple transfer incoming signals to all outputs

--> will allow to control hidden devices
--> instead of stuff like http://www.kabelstore.be/product/264308/category-209862/marmitek-invisible-control-4-remote-extender.html

**Comment by FDelporte** [comment]
could a receiver like this be used? has a nice housing so no need to do extra work...
http://www.amazon.com/Infrared-Receiver-Extender-Description-Compatibility/dp/B002JSDHCY
Tip: VCC 5V 
Center: 'Data' 
sleeve: 'ground'

**Comment by wouterverweirder** [comment]
Yeah, looks like this is just a regular IR led (which I'm using now) in a nicer housing.

**Comment by wouterverweirder** [comment]
As for receivers: the only time I use a receiver is when "copying" the original IR codes to my raspberry pi. I don't use any remotes anymore now, just my smartphone which triggers the recorded IR commands.

**Comment by FDelporte** [comment]
i could imagine using a permanent receiver to act as the passthrough for the unprogrammed buttons of all the remotes you only need now and then...
